seata-spring-boot-starter/src/main/java/io/seata/spring/boot/autoconfigure/properties/registry/RegistryRedisProperties.java

```diff
@@ -0,0 +1,50 @@
+/*
+ *  Copyright 1999-2019 Seata.io Group.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package io.seata.spring.boot.autoconfigure.properties.registry;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+import static io.seata.spring.boot.autoconfigure.StarterConstants.REGISTRY_REDIS_PREFIX;
+
+/**
+ * @author xingfudeshi@gmail.com
+ * @date 2019/10/03
+ */
+@Component
+@ConfigurationProperties(prefix = REGISTRY_REDIS_PREFIX)
+public class RegistryRedisProperties {
+    private String serverAddr = "localhost:6379";
+    private String db = "0";
```

Comment:
the type use int?

Reply:
The original type is string.
![image](https://user-images.githubusercontent.com/24457941/66636477-dad70780-ec43-11e9-8bda-1a9317e0e21b.png)